satpy/etc/readers/aws1_mwr_l1c_nc.yaml

```
@@ -0,0 +1,372 @@
+        file_type: aws_l1c_nc
+        file_key: data/calibration/aws_toa_brightness_temperature
+    '18':
+        name: '18'
+        frequency_double_sideband:
+           central: 325.15
+           side: 4.1
+           bandwidth: 1.8
+           unit: GHz
+        polarization: 'QV'
+        resolution: 10000
+        calibration:
+          brightness_temperature:
+            standard_name: toa_brightness_temperature
+        coordinates: [longitude, latitude]
+        file_type: aws_l1c_nc
+        file_key: data/calibration/aws_toa_brightness_temperature
+    '19':
+        name: '19'
+        frequency_double_sideband:
+           central: 325.15
+           side: 6.6
+           bandwidth: 2.8
+           unit: GHz
+        polarization: 'QV'
+        resolution: 10000
+        calibration:
+          brightness_temperature:
+            standard_name: toa_brightness_temperature
+        coordinates: [longitude, latitude]
+        file_type: aws_l1c_nc
+        file_key: data/calibration/aws_toa_brightness_temperature
+
+# --- Coordinates ---
+
+    longitude:
+        name: longitude
+        file_type: aws_l1c_nc
+        standard_name: longitude
+        units: degrees_east
+        file_key: data/navigation/aws_lon
+
+    latitude:
+        name: latitude
+        file_type: aws_l1c_nc
+        standard_name: latitude
+        units: degrees_north
+        file_key: data/navigation/aws_lat
+
+# --- Navigation data ---
+
+    solar_azimuth:
+        name: solar_azimuth
+        file_type: aws_l1c_nc
+        file_key: data/navigation/aws_solar_azimuth_angle
+        standard_name: solar_azimuth_angle
+        coordinates:
+            - longitude
+            - latitude
+
+    solar_zenith:
+        name: solar_zenith
+        file_type: aws_l1c_nc
+        file_key: data/navigation/aws_solar_zenith_angle
+        standard_name: solar_zenith_angle
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth:
+        name: satellite_azimuth
+        file_type: aws_l1c_nc
+        file_key: data/navigation/aws_satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith:
+        name: satellite_zenith
+        file_type: aws_l1c_nc
+        file_key: data/navigation/aws_satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        coordinates:
+            - longitude
+            - latitude
+
+file_types:
+    aws_l1c_nc:
+        file_reader: !!python/name:satpy.readers.mwr_l1c.AWS_MWR_L1CFile
+        file_patterns: [
+        'W_XX-OHB-Stockholm,SAT,{platform_name}-MWR-1C-RAD_C_OHB_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc',
+        'W_XX-OHB-Stockholm,SAT,{platform_name}-MWR-1C-RAD_C_OHB__{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc',]
```

Patterns have OHB and Stockholm hardcoded.
```suggestion
        'W_XX-{organization:s}-{location:s},SAT,{platform_name}-MWR-1C-RAD_C_{organization2}_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc',
        'W_XX-{organization:s}-{location:s},SAT,{platform_name}-MWR-1C-RAD_C_{organization2:s}__{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc',]
```

---

Fixed